covidthailand.py

```diff
@@ -3467,156 +3876,208 @@ def save_plots(df):
     ####################
 
     fig, ax = plt.subplots(figsize=[20, 10])
-    df['Deaths Age Median (MA)'] = df['Deaths Age Median'].rolling("7d").mean()
-    df['Deaths Age Min (MA)'] = df['Deaths Age Min'].rolling("7d").mean()
-    df['Deaths Age Max (MA)'] = df['Deaths Age Max'].rolling("7d").mean()
-    ax.set_ylim(bottom=0,top=100)
-    df["2021-04-01":].plot.line(
```

Comment:
it looks like I'm using mid dec is _2 time period and then some graphs use mid feb for some, which is not 3rd or 2nd wave? and here its april which is 3rd wave? so need something consisten with the time period names?

Reply:
what if we go with this

_0 = all,  (or stick to _all as it is now?)
_1 = 2020 up to mid year ie 1st wave.
_2 = mid dec to now. 2nd wave (can cut it off later)
_2.5 = mid feb to now - mainly for vaccinations?
_3 = 1st april. 3rd wave.

Reply:
Let's use `_30d` for the last 30 days. The rest is fine. We can reserve `_m` for a month in progress chart (if we ever need it).

Reply:
`_all` would be better than `_0`

Reply:
We should use `_25` to avoid the extra `.` in the file name (example to avoid: `positivity_2.5.png`)

Reply:
_all, _1, _2, _3, 30d then!

Reply:
`_30d` and `_25`